mscore/scoretreeview.h

```diff
@@ -0,0 +1,42 @@
+//=============================================================================
+//  MuseScore
+//  Music Composition & Notation
+//
+//  Copyright (C) 2020 MuseScore BVBA
+//
+//  This program is free software; you can redistribute it and/or modify
+//  it under the terms of the GNU General Public License version 2
+//  as published by the Free Software Foundation and appearing in
+//  the file LICENCE.GPL
+//=============================================================================
+
+#ifndef __STREEVIEW_H__
+#define __STREEVIEW_H__
+
+#include <QtWidgets>
+#include "musescore.h"
+#include "libmscore/score.h"
+#include "scoreitemmodel.h"
+
+namespace Ms {
+
+class ScoreTreeView : public QObject
```

Comment:
Why not inherit from QDockWidget and avoid the need for a `_dock` member?

Also, best rename this class `ScoreTreeWidget` to avoid confusion since the class itself is not a tree view; it is a widget that contains a tree view. If you inherit from `QTreeView` then you could call your class `ScoreTreeView`.